apps/condo/domains/common/components/Comments/CommentForm.tsx

```diff
@@ -8,13 +8,14 @@ import styled from '@emotion/styled'
 import { SendMessage } from '../icons/SendMessage'
 import { colors } from '@condo/domains/common/constants/style'
 import { useState } from 'react'
+import { InputWithCounter } from '../InputWithCounter'
 
 const Holder = styled.div`
   position: relative;
   button.ant-btn {
     position: absolute;
     right: 4px;
-    bottom: 4px;
+    bottom: 23px;
```

Comment:
Oo

Is it's all ok here? Looks strange

Reply:
This is because the button is positioned absolute to the Holder, and the InputWithCounter adds height to the Holder component.
4px would look like this:
<img width="255" alt="Снимок экрана 2021-07-21 в 20 34 45" src="https://user-images.githubusercontent.com/52532264/126517079-3dbe294c-fe3a-43c2-99aa-92ec1688f463.png">